### PR TITLE
Adding Desperate Amateurs

### PR DESCRIPTION
**Pull request by killjoy4115**

### CODE REVIEW

File: scrapers/DesperateAmateurs.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - desperateamateurs.com/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="title_bar"]/text()
```
Comment (bnkai) [inline scrapers/DesperateAmateurs.yml:11]: the title selector needs to  be restricted a bit 
it matches the `individual clips` and `updates` also if you check the log
```suggestion
        selector: (//div[@class="title_bar"])[1]
```
should be ok

File: scrapers/DesperateAmateurs.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      Tags:
+        Name: //a[@class="model_category_link"][contains(@href, 'category.php')]/text()
+      Image:
+        selector: //img[@class="thumbs"]/@src
+        postProcess:
+          - replace:
+            - regex: (.+image:\s+")(.+jpg)(.+)
+              with: $2
+            - regex: ^
+              with: "https://desperateamateurs.com/"
```
Comment (bnkai) [inline scrapers/DesperateAmateurs.yml:24]: the image has the same issue with above it matches all the thumbnails / 16 images
using this selector instead
```yaml
        selector: //img[@class="thumbs" and contains(src,"001.jpg")]/@src
```
only matches the first thumbnail
Another simpler alternative is using
```yaml
      Image: //meta[@name="twitter:image"]/@content
```
or 
```yaml
      Image: meta[@property="og:image"]/@content
```
Note that the meta use a different image each and usually not the same with the video player ( but neither does the 1rst thumb)

File: scrapers/DesperateAmateurs.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      Studio:
+        Name:
+          fixed: Desperate Amateurs
+      Date:
+        selector: //td[@class="date"]/text()
```
Comment (bnkai) [inline scrapers/DesperateAmateurs.yml:29]: same issue with the date
the selector matches the dates from the relevant clips
```suggestion
        selector: //table[@width="100%"]//td[@class="date"]/text()
```
Only matches the correct one

File: scrapers/DesperateAmateurs.yml (hunk 2 above)
Comment (killjoy4115) [reply scrapers/DesperateAmateurs.yml]: This makes sense. Thanks so much for checking this over.

**Pull request merged**